build.sbt

```diff
@@ -21,7 +21,8 @@ val collections =
       name := "collection-strawman",
       libraryDependencies ++= Seq(
         "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0",
-        "com.novocode" % "junit-interface" % "0.11" % Test
+        "com.novocode" % "junit-interface" % "0.11" % Test,
+        "org.scalatest" %% "scalatest" % "3.0.1" % Test
```

Comment:
I moved the code from another of my repos: https://github.com/pathikrit/scalgos/commit/bd1d794ffb573a563f155ff845bdedcf453542bd and thus it dragged in scalatest - I am fine porting tests to junit